build-tools/scripts/PrepareWindows.targets

```diff
@@ -4,7 +4,8 @@
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <_TopDir>$(MSBuildThisFileDirectory)..\..</_TopDir>
     <_XAPrepareExe>$(MSBuildThisFileDirectory)..\xaprepare\xaprepare\bin\$(Configuration)\xaprepare.exe</_XAPrepareExe>
-    <_XAPrepareStandardArgs>--no-emoji --run-mode=CI</_XAPrepareStandardArgs>
+    <_XAPrepareStandardArgs>--no-emoji --run-mode=CI -a</_XAPrepareStandardArgs>
```

Comment:
Wait, I missed that, do we need `-a` or not?

Reply:
The `-a` arg should be included: https://github.com/xamarin/xamarin-android/blob/6f238a2b98a4e03b9f8f7a9b84e865c1b515d036/build-tools/xaprepare/xaprepare/Main.cs#L101